src/megaclient.cpp

```diff
@@ -3565,6 +3566,8 @@ void MegaClient::locallogout()
     ssrs_enabled = false;
     nsr_enabled = false;
     aplvp_enabled = false;
+    smsve_state = -1;
+    sms_verifiedphone.clear();
```

Comment:
Always remember to clear any new variable related to the state of the account upon logout. In this case, the verified phone was kept ;)

Reply:
good point - perhaps we should arrange for that to happen automatically as it's an easy one to miss (for me, anyway).   Maybe a function that is called to initialize those values on both startup and on logout/reset (but also leaving aside any that really should be kept).   Or perhaps assign a default-constructed version of the MegaClient (but preserve anything that should not be overridden)

Reply:
I think it's fine as it is right now. The function you suggest is actually `init()`, which cleans anything related to the account's state and re-initializes the `MegaClient`. It is called during initialization and also as part for the `locallogout()` :)